From: Hailey Mothershead <hailmo@amazon.com>
Date: Tue, 6 Jun 2023 19:39:51 +0000
Subject: crypto: testmgr - Remove xts4096(paes) and xts512(paes)

As of commit a93492c `crypto: ccree - remove data unit size support`
These algorithms are not supported in the kernel.
---
 crypto/testmgr.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/crypto/testmgr.c b/crypto/testmgr.c
--- a/crypto/testmgr.c
+++ b/crypto/testmgr.c
@@ -5715,14 +5715,6 @@ static const struct alg_test_desc alg_test_descs[] = {
 		}
 	}, {
 #endif
-		.alg = "xts4096(paes)",
-		.test = alg_test_null,
-		.fips_allowed = 1,
-	}, {
-		.alg = "xts512(paes)",
-		.test = alg_test_null,
-		.fips_allowed = 1,
-	}, {
 		.alg = "xxhash64",
 		.test = alg_test_hash,
 		.fips_allowed = 1,

